tests/integration/issues/github_2992/test_flow.py

```diff
@@ -0,0 +1,17 @@
+from jina import Document, Flow
+from jina.types.arrays.memmap import DocumentArrayMemmap
+
+
+def test_dam_flow(tmpdir):
```

Comment:
use return_results or mock as in other tests

Reply:
is it okay now ? Or you were rejecting the usage of on_always ?